### DISCUSSION TITLE
可以在主界面更换微软账户和离线账户

### DISCUSSION THREAD

**Discussion by luoyuqi11** [Meloong-Git/PCL#526] [提问] [answered]
可以新增微软账户和离线账户

**Comment by WForst-Breeze** [comment, marked as answer]
稍微有点没理解您的意思，我就先按我的理解解释了：
1. 离线账户与微软账户的添加与切换可以在启动页面完成
2. 微软账户的添加与更换：待制作，请见 #43 ；目前想进行添加与切换操作需要手动退出登录后再进行登录
3. 离线账户的添加与更换：在启动界面单击“离线”选项进入离线登录界面，在下方输入框输入您想要的离线登录用户名。如想快捷更换离线账户，在输入框右侧您会看到一个类似v字的小按钮，单击它，此时输入框展开，下方列出的即是您使用过的所有离线账户名称（需至少使用该名字进入任意版本一次才会被记录）

**Reply by WForst-Breeze** [reply]
如果我理解错了很抱歉qwq，我理解能力也不是很好，如果有帮到你最好啦~
